reactioncommerce/reaction
# Fix issue dragging images in PDP when settings drawer is open

This resolves #2478

### Steps to test
1. Login as an admin
1. Click on the edit icon for the variant so that the edit drawer opens
1. Drag an image into the "main" image area (over the + sign)
1. Observe that image is now uploaded for the first variant
1. Publish
1. Observe that it shows up in the product grid.
